libllvm-pdb-wrapper/wrapper.cpp

```diff
@@ -189,14 +200,26 @@ pdb_file::add_function_symbol(const char *name, uint16_t section_index, uint32_t
 
 void pdb_file::add_function_symbol(const char *name, uint16_t section_index, uint32_t section_offset) {
     auto &GsiBuilder = m_pdb_builder->getGsiBuilder();
+#ifdef LLVM_NEWER
+    auto symbol = llvm::pdb::BulkPublic();
+#else
     auto symbol = PublicSym32(SymbolKind::S_PUB32);
+#endif
 
     symbol.Name = name;
+#ifdef LLVM_NEWER
+    symbol.Flags |= static_cast<uint16_t>(PublicSymFlags::Function);
+#else
     symbol.Flags |= PublicSymFlags::Function;
+#endif
+
     symbol.Segment = section_index;
     symbol.Offset = section_offset;
-
+#ifdef LLVM_NEWER
+    m_PublicsSyms.push_back(symbol);
+#else
     GsiBuilder.addPublicSymbol(symbol);
```

Comment:
llvm 13 does not have a function to individually add public symbols to the pdb only to add them in bulk using BulkPublic